compiler/rustc_mir_transform/src/lib.rs

```diff
@@ -130,7 +131,7 @@ pub fn provide(providers: &mut Providers) {
         mir_generator_witnesses: generator::mir_generator_witnesses,
         optimized_mir,
         is_mir_available,
-        is_ctfe_mir_available: |tcx, did| is_mir_available(tcx, did),
+        is_ctfe_mir_available: is_mir_available,
```

Comment:
This change is unrelated. Could you submit it as a separate PR?

Reply:
i removed this for now, i'll open a PR soon.